scripts/src/models/all.ts

```diff
@@ -1,83 +1,111 @@
+import { Model } from "sequelize";
+type GenericModel = Model<any, any>;
```

Comment:
why do you need this?

Reply:
Because they are using older version of typescript where you couldn't specify default generics constraints.  
So the definition for Model is:
```
interface Model<TInstance, TAttributes> extends Hooks<TInstance>, Associations { ... }
```
So you **have** to specify the generic constraints, and so instead of keep writing: `Model<any, any>` everywhere I created this type alias.

Now days you can do:
```
interface Model<TInstance = any, TAttributes = any> extends Hooks<TInstance>, Associations { ... }
```